certified-connectors/Airmeet/apiDefinition.swagger.json

```diff
@@ -599,7 +608,7 @@
           {
             "name": "allAirmeets",
             "in": "query",
-            "required": false,
+            "required": true,
```

Comment:
@amit-vb The parameter has been changed from optional to required. This change may break existing flows.
can you please confirm that no users will be affected by this change?

Reply:
Hi @Amjed-Ayoub 
right now we have only internal users & so after the new version will send them a notification to use the latest versions.